please.sh

```diff
@@ -761,6 +761,17 @@ build_and_test_64 () {
 	test $# = 0 ||
 	die "Expected no argument, got $#: %s\n" "$*"
 
+	clean_t='perl -pe "s/ok \d+ # skip/skipped:/" |
+			 perl -pe "s/not ok \d+ -(.*)# TODO known breakage/known e:$1/" |
+			 perl -pe "s/\*\*\* (.+) \*\*\*/$1/g" |
```

Comment:
Rather than having plenty of `perl` processes piping into each other (which can cause inexplicable hangs on Windows), it should be better to call something like:

```sh
perl -pe 's/ok \d+ # skip/skipped:/;
    s/not ok \d+ -(.*)# TODO known breakage/known e:$1/;
    s/\*\*\* (.+) \*\*\*/$1/g;
[...]'
```

This also has the advantage of not needing `$` to be escaped, as we are now using single quotes, not double quotes.